apis/supernova/src/hooks.js

```diff
@@ -1064,14 +1064,28 @@ export function useRenderState() {
   };
 }
 
+/**
+ * Blur function
+ * @name BlurFunction
+ * @function
+ * @param {boolean} blurState parameter to indicate blur state
+ */
+
+/**
+ * Focus Selection function
+ * @name FocusSelectionFunction
+ * @function
```

Comment:
you can use @callback to set both name and that it is a function type definition in one line
```suggestion
 * @callback FocusSelectionFunction
```